### ISSUE TITLE
500 Error when deleting an interface connected to a circuit via REST API

### ISSUE THREAD

**Issue by mngan** [nautobot/nautobot#1723]
### Environment
* Python version: 3.8
* Nautobot version: 1.2.11

### Steps to Reproduce
1. Create a device
2. Create an interface
3. Create a circuit
4. Connect interface to circuit
5. Delete interface via REST API (DELETE /api/dcim/interfaces/b7ca263e-237d-52e4-8369-d82c708eed60/)

This maybe related to #1694 

### Expected Behavior

The interface should get deleted, and the circuit terminator should get NULL'ed. 

### Observed Behavior

```
19:20:46.244 DEBUG   nautobot.core.api.views.ModelViewSet views.py                  get_serializer_class() :
  Using serializer <class 'nautobot.dcim.api.serializers.InterfaceSerializer'>
19:20:46.537 INFO    nautobot.core.api.views.ModelViewSet views.py                       perform_destroy() :
  Deleting interface et-0/1/8:2 (PK: b7ca263e-237d-52e4-8369-d82c708eed60)
19:20:46.588 DEBUG   nautobot.dcim.cable  signals.py         nullify_connected_endpoints() :
  Nullifying termination A for cable #21eb1df9-723a-5643-8637-d7d1e97ffa2a
Internal Server Error: /api/dcim/interfaces/b7ca263e-237d-52e4-8369-d82c708eed60/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/nautobot/core/api/views.py", line 193, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/mixins.py", line 91, in destroy
    self.perform_destroy(instance)
  File "/usr/local/lib/python3.8/site-packages/nautobot/core/api/views.py", line 252, in perform_destroy
    return super().perform_destroy(instance)
  File "/usr/local/lib/python3.8/site-packages/rest_framework/mixins.py", line 95, in perform_destroy
    instance.delete()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 947, in delete
    return collector.delete()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/deletion.py", line 435, in delete
    signals.post_delete.send(
  File "/usr/local/lib/python3.8/site-packages/django/dispatch/dispatcher.py", line 177, in send
    return [
  File "/usr/local/lib/python3.8/site-packages/django/dispatch/dispatcher.py", line 178, in <listcomp>
    (receiver, receiver(signal=self, sender=sender, **named))
  File "/usr/local/lib/python3.8/site-packages/nautobot/dcim/signals.py", line 172, in nullify_connected_endpoints
    instance.termination_a.save()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 753, in save
    self.save_base(using=using, force_insert=force_insert,
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 801, in save_base
    post_save.send(
  File "/usr/local/lib/python3.8/site-packages/django/dispatch/dispatcher.py", line 177, in send
    return [
  File "/usr/local/lib/python3.8/site-packages/django/dispatch/dispatcher.py", line 178, in <listcomp>
    (receiver, receiver(signal=self, sender=sender, **named))
  File "/usr/local/lib/python3.8/site-packages/nautobot/circuits/signals.py", line 54, in update_connected_terminations
    rebuild_paths_circuits(peer)
  File "/usr/local/lib/python3.8/site-packages/nautobot/circuits/signals.py", line 28, in rebuild_paths_circuits
    invalidate_obj(cp.origin)
  File "/usr/local/lib/python3.8/site-packages/cacheops/invalidation.py", line 35, in invalidate_obj
    model = obj.__class__._meta.concrete_model
AttributeError: type object 'NoneType' has no attribute '_meta'
[pid: 49|app: 0|req: 2/2] 172.30.10.1 () {42 vars in 694 bytes} [Mon May  2 19:20:46 2022] DELETE /api/dcim/interfaces/b7ca263e-237d-52e4-8369-d82c708eed60/ => generated 30434 bytes in 173 msecs (HTTP/1.0 500) 8 headers in 298 bytes (1 switches on core 0)
```

**Comment by gsnider2195** [comment]
Need to confirm this is still an issue after #1694 was resolved.

**Comment by bryanculver** [comment]
Confirmed no longer an issue as of 1.5.4, likely fixed in 1.3.9 with #2050.
